docs/resources/ec_deployment.md

```diff
@@ -197,55 +199,55 @@ The optional `kibana.config` and `kibana.topology.config` blocks support the fol
 
 #### APM
 
-The optional `apm` block supports the following:
+The optional `apm` block supports the following settings:
 
-* `topology` - (Optional) Topology element which can be set multiple times to compose complex topologies.
-* `elasticsearch_cluster_ref_id` - (Optional) This field references the ref_id of the deployment Elasticsearch cluster, it is best left to the default value (Defaults to `main-elasticsearch`).
-* `ref_id` - (Optional) ref_id to set on the APM resource. It is best left to the default value (Defaults to `main-apm`).
-* `config` (Optional) APM settings which will be applied to all topologies unless overridden on the topology element. 
+* `topology` - (Optional) Can be set multiple times to compose complex topologies.
+* `elasticsearch_cluster_ref_id` - (Optional) This field references the `ref_id` of the deployment Elasticsearch cluster. The default value `main-elasticsearch` is recommended.
+* `ref_id` - (Optional) Can be set on the APM resource. The default value `main-apm` is recommended.
+* `config` (Optional) APM settings applied to all topologies unless overridden in the `topology` element. 
 
 ##### Topology
 
-The optional `apm.topology` block supports the following:
+The optional `apm.topology` block supports the following settings:
 
-* `instance_configuration_id` - (Optional) Instance Configuration ID from the deployment template. By default, it will use the deployment template default instance configuration, but it can be changed. See top level note on regions and deployment templates. There's no need to change this value since Kibana only has one _instance type_.
-* `size` - (Optional) Amount of memory (RAM) per topology element in the "<size in GB>g" notation. When omitted, it will default to the deployment template value.
-* `size_resource` - (Optional) Type of resource the size is being assigned to. (Defaults to `"memory"`).
-* `zone_count` - (Optional) Number of zones that the APM deployment will span. This is used to set HA. When omitted, it will default to the deployment template value.
-* `config` (Optional) APM settings which will be applied at the topology level. 
+* `instance_configuration_id` - (Optional) Default instance configuration of the deployment template. No need to change this value since Kibana has only one _instance type_.
```

Comment:
```suggestion
* `instance_configuration_id` - (Optional) Default instance configuration of the deployment template. No need to change this value since APM has only one _instance type_.
```